Dockerfile

```diff
@@ -7,3 +8,4 @@ RUN /bin/herokuish buildpack install \
 	&& ln -s /bin/herokuish /exec
 COPY include/default_user.bash /tmp/default_user.bash
 RUN bash /tmp/default_user.bash && rm -f /tmp/default_user.bash
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
don't delete `/tmp/*`. that's where we bundle the buildpacks

https://github.com/gliderlabs/herokuish/blob/be0e139f66a7fa182715d79220fd1f4bc3140075/include/herokuish.bash#L13

Reply:
ah dangit. that was @pynixwang's edit, i'll fix it.

Reply:
Still need to change this line